testsuite/model/src/main/java/org/keycloak/testsuite/model/RequireProvider.java

```diff
@@ -41,4 +41,11 @@
      */
     String[] only() default {};
 
+    /**
+     * Specifies provider IDs that does not satisfy this requirement. In other words, there must be another provider
+     * of type {@code value()} for satisfying this requirement. Can be used together with only, however it does
+     * not make a lot of sense to exclude some of
```

Comment:
Minor: this javadoc sentence looks unfinished.

Reply:
Thanks, fixed.